### DISCUSSION TITLE
What tool would you like to use to generate metadata?

### DISCUSSION THREAD

**Discussion by code-by-justin** [Security-Onion-Solutions/securityonion#2087] [Unsupported Versions] [answered]
Hello,

I have been searching  through documentation, this forum, as well as watching the Security Onion Essentials videos passed initial installation. 

During the installer for a Distributed - ManagerSearch node it asks

"What tool would you like to use to generate metadata?"
Zeek
Suricata
![image](https://user-images.githubusercontent.com/24820491/100187182-934a2400-2eb5-11eb-9a80-7e1ed3e393aa.png)

Is there any advantages or disadvantages between them using either option?

I haven't been able to find any info on the difference advantages/disadvantages of using either option.

Is this outlined anywhere? 

I set up a Manager + Search for testing using Suricata for metadata (which was not the default option) but would like to choose the best option for the upcoming Production Set-up.

Thanks.

**Comment by weslambert** [comment, marked as answer]
It's really your preference -- at this time, I believe Zeek provides more logging around different protocols, so many folks may choose Zeek, but you can do either.
